server/region_syncer/server.go

```diff
@@ -42,6 +42,11 @@ const (
 	defaultHistoryBufferSize = 10000
 )
 
+var (
+	// ErrNotStarted is returned when current server is starting.
+	ErrNotStarted = status.Errorf(codes.Unavailable, "server not started")
```

Comment:
How about reducing the duplicated definision?

Reply:
I didn't think of a good way. Maybe only a normal error is returned in region_syncer?